### ISSUE TITLE
Add option not to enlarge images

### ISSUE THREAD

**Issue by matylla** [lovell/sharp#36]
Possible options:

```
sharp('file.jpg', {enlarge: false}).resize(100, 100)
```

```
sharp('file.jpg').resize(100, 100).enlarge(false)
```

```
sharp('file.jpg').resize(100, 100).noEnlarge()
```

Say I want to resize 1000s of images to fit 100x100 but some of those are already 50x50. Than it would be nice not to enlarge those tiny images (quality degradation) and return the same input image in a callback.

**Comment by OZZlE** [comment]
> This is now in master and will be published with v0.5.0 - see https://github.com/lovell/sharp#withoutenlargement

I cannot find this in any documentation 'withoutEnlargement' or anything similar?

**Comment by OZZlE** [comment]
Sorry it was here my bad http://sharp.pixelplumbing.com/en/stable/api-resize/
